readthedocs/search/management/commands/reindex_elasticsearch.py

```diff
@@ -236,6 +235,13 @@ def handle(self, *args, **options):
         update_from = options['update_from']
         if change_index:
             timestamp = change_index
+            print(
+                f'You are about to change change the index from {models} to `[model]_{timestamp}`',
+                '**The old index will be deleted!**'
+            )
+            if input('Continue? y/n') != 'y':
```

Comment:
I think we probably want this to make the input more obvious, but not 100% sure how `input` works. 

```suggestion
            if input('Continue? y/n: ') != 'y':
```

Reply:
ah, you are right, currently the answer will be next to the question